app-admin/etcd-wrapper/files/etcd-wrapper

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -e
+
+USAGE="
+Usage: $0 [etcd flags]
+
+Wrapper for launching etcd via rkt.
+
+The version of etcd to be used can be set through ETCD_VERSION. The version
+must be set to one of the image tags published here:
+
+  https://quay.io/repository/coreos/etcd?tab=tags
+
+Alternatively, the keywords 'alpha', 'beta', and 'stable' may be used to specify
+the version. If no version is provided, the CoreOS update group is used.
+"
+
+case ${1} in
+	-h | --help)
+		echo "${USAGE}"
+		exit 0
+		;;
+esac
+
+source /usr/share/coreos/update.conf
+source /etc/coreos/update.conf
+
+ETCD_VERSION="${ETCD_VERSION:-${GROUP:-stable}}"
+ETCD_ACI="${ETCD_ACI:-coreos.com/etcd}"
+
+case ${ETCD_VERSION} in
```

Comment:
The updates to etcd, or any other containerized system service, doesn't have to depend on the CoreOS updates. I imagine there to be a default version per CoreOS version, but the user, or the container updater service, should be able to update it independently. In an OOB discussion with @crawford about this, the implementation we had in mind consists of targets for component groups (e.g. kubelet + docker) which uses service templates to adjust the component version. Details will go in a design doc as this is not specific to etcd.